group_vars/all/vlans.yaml

```diff
@@ -1,17 +1,326 @@
-    name: "VLAN 100"
-    description: "This is the description of example vlan 100"
+  10:
+    name: "CSAIL-MAIN"
+    description: "Openshift public facing services (horizon, keystone)"
     switches:
-      - oct_tors
-  101:
-    name: "VLAN 101"
-    description: "This is the description of example vlan 101"
+      - oct_data
+  57:
+    name: "UMA-57"
+    description: "UMass VLAN 57 Public IPs for Chameleon"
     switches:
-      - oct_tors
-  102:
-    name: "VLAN 102"
-    description: "hello world"
+      - oct_data
+  84:
+    name: "UMA-84"
+    description: "UMass VLAN 84 Public IPs for CloudLab"
+    switches:
+      - oct_data
+  86:
+    name: "UMA-86"
+    description: "UMass VLAN 86 Public IPs for Fabric"
+  105:
+    name: "MOC-BU-PUBLIC"
+    description: "Public IPs from BU"
+    switches:
+      - oct_data
+  127:
+    name: "MOC-NEU-PUBLIC"
+    description: "NEU Public IP for infrastructure"
+    switches:
+      - oct_data
+  201:
+    name: "MOC-FOREMAN"
+    description: "Foreman Provisioning. SNMP for OpenStack and Ceph"
+    switches:
+      - oct_data
+  202:
+    name: "MOC-OPENSTACK-INTERNAL"
+    description: "OpenStack internal API"
+    switches:
+  203:
+    name: "MOC-OPENSTACK-TENANT"
+    description: "OpenStack tenant network"
+    switches:
+  204:
+    name: "MOC-GENERAL"
+    description: "Intranet (routable to internet). SNMP for client nodes."
+    switches:
+      - oct_data
+  205:
+    name: "MOC-GLUSTER"
+    description: "Gluster/VM migration - oVirt"
+    switches:
+  206:
+    name: "MOC-OPENSTACK-ISOLATION"
+    description: "OStack isolation native vlan for trunk only ports"
+    switches:
+  207:
+    name: "MOC-SWITCH-MGMT"
+    description: "For OCT/UMass Switch Management"
+    switches:
+      - oct_data
+      - oct_mgmt
+  208:
+    name: "ESI-CONTROL-PLANE"
+    description: "ESI control plane"
+    switches:
+      - oct_data
+  209:
+    name: "OCP-PROD-PROVISIONING"
+    description: "Openshift Internal (Baremetal 4.x)"
+    switches:
+      - oct_data
+  210:
+    name: "MOC-NFS-ZERO-CLUSTER"
+    description: "NFS for zero cluster"
+    switches:
+      - oct_data
+  211:
+    name: "MOC-NESE"
+    description: "New England Storage Exchange (NESE)"
+    switches:
+      - oct_data
+  212:
+    name: "OCP-STAGING-INTERNAL"
+    description: "Openshift Staging Internal (ocp-staging)"
+    switches:
+  213:
+    name: "MOC-PROD-CEPH-ISCSI"
+    description: "Ceph Cluster iSCSI"
+    switches:
+      - oct_data
+  249:
+    name: "CEPH-INTERNAL"
+    description: "Ceph Cluster (internal)"
+    switches:
+  250:
+    name: "MOC-PROD-CEPH"
+    description: "Ceph public (for clients)"
+    switches:
+      - oct_data
+  252:
+    name: "MOC-RESEARCH-CEPH"
+    description: "MOC Research Ceph"
+    switches:
+      - oct_data
+  259:
+    name: "STAGING-FOREMAN"
+    description: "Staging - Foreman"
+    switches:
+  267:
+    name: "MOC-RESEARCH-CEPH-EXTRA"
+    description: "****"
```

Comment:
"MOC Research ceph students"
Some PhD students working with research ceph have been using that VLAN.